src/main/java/noobconversation/Conversation.java

```diff
@@ -0,0 +1,74 @@
+package noobconversation;
+
+import format.Formatting;
+import task.Task;
+
+import java.util.Scanner;
+
+public class Conversation {
+
+    protected Formatting f;
+
+    public Conversation(){
+        f = new Formatting();
+    }
+
+    public void communicate() {
```

Comment:
this is okay but it would be more descriptive if you used somethign like `startConversation()` instead.